CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 ### Enhancements
 
+* Added a RealmModel interface that can be used instead of extending RealmObject.
```

Comment:
also "Removed  deprecated method RealmObject.removeFromRealm() from public API." in Breaking changes.